app/cdap/components/NamespaceAdmin/ServiceAccounts/EditConfirmDialog.tsx

```diff
@@ -63,13 +63,13 @@ const StyledTextField = styled(TextField)`
  * @return string, the gcloud cli command to run
  */
 const getGcloudCommand = ({
-  tenantProjectId = '${TENANT_PROJECT_ID}',
+  tenantProjectId = '${TENANT_PROJECT_ID}.svc.id.goog',
```

Comment:
nit: let's call the variable name here as well `k8sWorkloadIdentityPool` instead of `tenantProjectId`.

Reply:
done